services/worker-manager/config.yml

```diff
@@ -19,6 +19,7 @@ defaults:
   postgres:
     readDbUrl: !env READ_DB_URL
     writeDbUrl: !env WRITE_DB_URL
+    dbCryptoKeys: !env:json:optional DB_CRYPTO_KEYS
```

Comment:
Makes sense. Do you also want me to change `DB_CRYPTO_KEYS` to `CRYPTO_KEYS`? I originally picked dbCryptoKeys to align myself with https://github.com/taskcluster/taskcluster/blob/master/ui/docs/manual/deploying/database.mdx#supporting-encrypted-columns.

Reply:
Ah, I see. The env var makes sense to have the `DB` prefix since it isn't nested. But I think having the parallel with the env var here is fine if you'd like. Leave it or change it, either is fine with me!